src/compiler.js

```diff
@@ -0,0 +1,32 @@
+import { resolve } from 'path';
+import promisify from 'es6-promisify';
+import rimraf from 'rimraf';
+import prerender from './lib/prerender';
+import webpackConfig from './lib/webpack-config';
+import runWebpack, { showStats, writeJsonStats } from './lib/run-webpack';
+
+export default env => {
+	let config = webpackConfig(env)
+
+	return {
+		async clean() {
+			let dest = resolve(env.cwd || process.cwd(), env.dest || 'build');
```

Comment:
In retrospective this could have been `config.output.path` - if this lands before #56 I'll change it there